src/panels/lovelace/views/hui-view.ts

```diff
@@ -94,11 +97,9 @@ export class HUIView extends UpdatingElement {
     }
 
     if (configChanged && !this._layoutElement) {
-      this._layoutElement = getLovelaceViewElement(
-        viewConfig.panel
-          ? PANEL_VIEW_LAYOUT
-          : viewConfig.layout || DEFAULT_VIEW_LAYOUT
```

Comment:
We still need to be backwards compatible with the old panel config format?

Reply:
It is compatible. It checks if the flag is set and runs the panel view instead